mqt/qcec/bindings.cpp

```diff
@@ -536,6 +553,7 @@ PYBIND11_MODULE(pyqcec, m) {
       .def_readwrite("application", &Configuration::application)
       .def_readwrite("functionality", &Configuration::functionality)
       .def_readwrite("simulation", &Configuration::simulation)
+      .def_readwrite("parameterized", &Configuration::parameterized)
```

Comment:
I can't link to the proper place in this review since there have been no changes there, but these options are not reflected in the docs.
A subpage needs to be added here https://github.com/cda-tum/qcec/blob/e9aeae06ee1c1cd30c759109df15c18ec2eaf476/docs/source/library/Configuration.rst
And the subpage should probably look similar to any in https://github.com/cda-tum/qcec/tree/symbolic/docs/source/library/configuration

Reply:
Done